index.html

```diff
@@ -189,10 +189,46 @@ <h2 class="about-primary-text">About me</h2>
             </ul>
           </div>
         </ul>
-      </div>
+      </div>  
 
     </div>
   </div>
 </section>
+
+
+
+
+
+<section class="contact">
+  <div class="bottom">
+<p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+<form action="https://formspree.io/f/mlevodye" method="POST" >
+<input type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required>
+<input type="email" id="mail" name="user_email" placeholder="Email address" required>
+<textarea type="text" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+<button type="submit" value="submit">Get in touch</button>
+</form>
+  </div>
+</section>
+
+<footer class="icons-home">
+   <div class="bottom-icons">
+    <ul class="social-icons">
+      <li><a href="#" id="github">
+          <img src="Images\bottom1.png" alt="github"></a></li>
+      <li><a href="#" id="linkedin">
+          <img src="Images\bottom2.png" alt="linkedin"></a></li>
+      <li><a href="#" id="angelist">
+          <img src="Images\bottom3.png" alt="angelist"></a></li>
+      <li><a href="#" id="twitter">
+          <img src="Images\bottom4.png" alt="twitter"></a></li>
+      <li><a href="#" id="medium">
+          <img src="Images\bottom5.png" alt="medium"></a></li>
+    </ul>
+   </div>
+     <div class="home">
+        <img src="Images\Shape.png" alt="home.png">
+     </div>
+</footer>
```

Comment:
- [ ] There seems to be an extending white space after the footer section on your webpage which is not supposed to be there. Kindly fix this.

**This is the white space I am talking about:**

[Screencast from 01-11-2022 04:10:28 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/199240904-f2e2de2c-c817-47c9-a81b-7cde3d4144aa.webm)